modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -34,9 +34,7 @@ export default class TileLayer extends CompositeLayer {
 
   get isLoaded() {
     const {tileset} = this.state;
-    return tileset.selectedTiles.every(
-      tile => tile.layers && tile.layers.every(layer => layer.isLoaded)
-    );
+    return tileset.isLoaded;
```

Comment:
Sublayers may have other async props that require loading?

Reply:
I thought those would be covered by `tile.isLoaded`, but maybe not. I reverted this.